tests/gpu/misc/hosts

```diff
@@ -1,2 +1,2 @@
 [rocker-server-aws]
-54.81.189.38 ansible_ssh_private_key_file=~/Projects/Rocker/Rocker-aws-1/rocker-gpu.pem ansible_ssh_user=ubuntu
```

Comment:
What does this IP address represent? It seems to change frequently. Does it make sense to specify it in a way that is not managed by git? (Including this file in gitignore and managing the template with git instead)

Reply:
Good point.  It's just for my Ansible build, so I'll add it to `.gitignore`